examples/intro/inducing_path.py

```diff
@@ -0,0 +1,111 @@
+"""
+======================================================
+An introduction to Inducing Paths and how to find them
+======================================================
+
+A path p is called an ``inducing path`` relateve to <L,S>
+on an ancestral graph, where every non-endpoint vertex on p
+is either in L or a collider, and every collider on p is an ancestor
+of either X , Y or a member of S.
+
+
+In other words, it is a path between two nodes that cannot be
+d-seperated, making it active regardless of what variables
+we condition on.
+"""
+
+import pywhy_graphs
+from pywhy_graphs import PAG
+from pywhy_graphs.viz import draw
+
+# %%
+# The Graph
+# ----------------------------------------------
+# To illustrate the workings of the inducing path algorithm, we will
+# construct the causal graph from figure 2 of https://arxiv.org/pdf/1104.5617.pdf.
+
+
+G = PAG()
```

Comment:
You want to draw the graph, not the PAG?

So there shouldn't be any circle edges since there is no such thing as an inducing path for PAGs, or CPDAGs.